### ISSUE TITLE
Remove feature toggle champva_confirmation_email_bugfix

### ISSUE THREAD

**Issue by stevelong00** [department-of-veterans-affairs/va.gov-team#97768]
AC
- [ ] Remove all references to feature toggle champva_confirmation_email_bugfix
- [ ] Remove temporary rubocop:disable statements related to this toggle
- [ ] Remove unit tests that cover when this toggle is disabled
- [ ] Keep and clean up unit tests that cover when this toggle is enabled
- [ ] Remove feature toggle champva_confirmation_email_bugfix

**Comment by cloudmagic80** [comment]
This is a duplicate ticket

**Comment by andrea-merrill** [comment]
@michaelclement do we need this after the story in this current sprint to remove any toggles older than 30 days?

**Comment by michaelclement** [comment]
I don't think we still need it. I'll link that ticket [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/104641) just so we're aware.

**Comment by andrea-merrill** [comment]
closing @ManOnTheMooney
